python-spec/src/somacore/data.py

```diff
@@ -195,8 +195,18 @@ def create(
         :param uri: The URI where the array will be created.
         :param type: The Arrow type to store in the array.
             If the type is unsupported, an error will be raised.
-        :param shape: The length of each dimension as a sequence,
-            e.g. ``(100, 10)``. All lengths must be in the postive int64 range.
+        :param shape: The maximum capacity of each dimension, including room
+            for any intended future appends, as a sequence.  E.g. ``(100, 10)``.
+            All lengths must be in the postive int64 range, or ``None``.
+
+            For `SOMASparseNDArray` only, if a slot is None, then the maximum
```

Comment:
<code>\`\`doubleticks\`\`</code> here and elsewhere

Reply:
derp

thanks for the copy-edit here @thetorpedodog ... i am forever forgetting to double these up 😞